uaclient/entitlements/tests/test_fips.py

```diff
@@ -109,6 +109,55 @@ def test_conditional_packages(
         conditional_packages = entitlement.conditional_packages
         assert expected == conditional_packages
 
+    @pytest.mark.parametrize(
+        "fips_version, assume_yes, expected_continue",
+        (
+            (
+                "0",
+                True,
+                True,
+            ),
+            (
+                "0",
+                False,
+                False,
+            ),
+            (
+                "999",
+                True,
+                True,
+            ),
+            (
+                "999",
+                False,
+                True,
+            ),
+        ),
+    )
+    @mock.patch(M_PATH + "apt.get_pkg_candidate_version")
+    @mock.patch(M_PATH + "util.prompt_for_confirmation")
+    def test_kernel_downgrade(
+        self,
+        m_prompt_for_confirmation,
+        m_pkg_candidate_version,
+        fips_version,
+        assume_yes,
+        expected_continue,
+        entitlement,
+    ):
+        """Test kernel downgrades block install if user denies prompt"""
+        m_prompt_for_confirmation = util.prompt_for_confirmation
```

Comment:
I think I am missing something. Why do we need this line here ?

Reply:
We do not!  Good catch, that was left over from my experiments for the best way to handle this mock.  I will clean it up and then we should be good to merge.